### DISCUSSION TITLE
Move test bundles to folder "tests"

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-equinox/p2#110] [Ideas]
A Structured layout is easier to build and easier to find what one is looking for, thus I like to propose that we move all test-bundles (that already use a separate test parent) to a "tests" folder, and move the test parent there so it is automatically discovered and we do not need to configure parent explicitly.

Any thought or concerns?

**Comment by HannesWell** [comment]
Sounds good to me. The test parent only defines two properties. For Test-Plugins that require them we could simply use corresponding pom.model.property... entires in the build.properties. Then we could also use the automatically generated aggregator, given that all tests run successfully a.t.m.

**Reply by HannesWell** [reply]
I want to add that we have the same situation in the equinox repo. So if we could discuss it for both.

**Comment by akurtakov** [comment]
I like the idea of having things more automated.
